ratelimit/tests.py

```diff
@@ -389,6 +391,20 @@ def do_increment(request):
         assert do_increment(req), 'Request should be rate limited.'
         assert not_increment(req), 'Request should be rate limited.'
 
+    def test_longer_window_than_deault_timeout(self):
+        @ratelimit(key='ip', rate='1/m', block=True)
+        def view(request):
+            return True
+
+        req = rf.get('/')
+
+        assert view(req), 'First request works.'
+
+        time.sleep(3)
```

Comment:
`time.sleep` in a test is rough, and given that we have to explicitly set the default (which is the thing that changed), this is not a great way to go about testing this.

A better approach is probably to mock the cache and assert that the cache functions are called with the appropriate timeout values (as I mentioned in https://github.com/jsocol/django-ratelimit/issues/85#issuecomment-141243154).